### DISCUSSION TITLE
How call command directly from dispatcher

### DISCUSSION THREAD

**Discussion by shurshilov** [aiogram/aiogram#1441] [Q&A]
Hi I use another wev server, and i want use aiogram.
I created Bot
![image](https://github.com/aiogram/aiogram/assets/11828278/43a1cd7c-1308-489f-b87c-01f2747783d6)
Also i added routers for dispacther like this 
![image](https://github.com/aiogram/aiogram/assets/11828278/dccb2baa-7e68-4138-abce-3d7ae13ffea8)
Now in my webhook i want that dicpatcher call command
![image](https://github.com/aiogram/aiogram/assets/11828278/053f510a-cd05-4d78-a312-a3aa90592fd5)
How can I do this?

**Comment by devkhan** [comment]
Similar: https://github.com/aiogram/aiogram/issues/1528

**Comment by devkhan** [comment]
Solution provided here: https://github.com/aiogram/aiogram/issues/1528#issuecomment-2197261886
